BlueprintUICommonControls/Sources/Image.swift

```diff
@@ -15,15 +15,21 @@ public struct Image: Element {
     /// not precisely match the size that the element is assigned.
     public var contentMode: ContentMode
 
+    /// iOS 14 added support for Image Descriptions using VoiceOver. This is not always appropriate.
+    /// Set this to `true` to prevent VoiceOver from discribing the displayed image.
+    public var blockAccessibilityDiscription: Bool
+
     /// Initializes an image element with the given `UIImage` instance.
     public init(
         image: UIImage?,
         contentMode: ContentMode = .aspectFill,
-        tintColor: UIColor? = nil
+        tintColor: UIColor? = nil,
+        blockAccessibilityDiscription: Bool = false
     ) {
         self.image = image
         self.contentMode = contentMode
         self.tintColor = tintColor
+        self.blockAccessibilityDiscription = blockAccessibilityDiscription
```

Comment:
I just noticed this is spelled wrong: `blockAccessibilityDiscription` should be `blockAccessibilityDescription`